Home_Task_9.py

```diff
@@ -0,0 +1,44 @@
+"""
+    Task: Operations with files, REs.
+
+    Description:
+
+write script that will read "input.txt" file
+and find there all characters from english alphabet
+collect these characters and their positions in file
+after info collected this info as text
+write to "output.txt" file in such format
+####################
+<first found letter> -> pos1
+<next_letter> -> pos2
+<next_letter -> pos3
+etc
+#######################
+"""
+
+import re
+
+try:
+#opening both input and output files
+    with open('input.txt', 'r') as fh, open('output.txt', 'w') as fh2:
+        input_data = fh.read()
+
+        # getting the list of letters matching the search criteria
+        letters_found = re.findall('[a-z]', string=input_data)
+
+        # getting matches as iterators
+        iterables_found = re.finditer('[a-z]', string=input_data)
```

Comment:
re.finditer already gives you smth aka m.group(0) to get current value ( letter) 
No sense in line #27 ( get all letters with findall)